File: FraseDoDia/app/src/main/java/com/myo/frasedodia/MainActivity.java
package com.myo.frasedodia;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

import java.util.Random;

public class MainActivity extends AppCompatActivity {

    String[] frases = {
            "A vida é bela, basta sorrir para ela!",
            "Disciplina é liberdade!",
            "Ter bondade é ter coragem!",
            "Compaixão é fortaleza!"
    };

    TextView tvFrase;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        tvFrase = findViewById(R.id.tvResultado);
    }

    public void gerarFrase(View view){

        int num = new Random().nextInt(3);

        tvFrase.setText(frases[num]);

    }

    public void exibirTodas(View view){

        String string = "";

        for(String frase : frases){

            // Saída na sysout
            /*System.out.println(frase);*/

            // Saída na tv da tela

            string = string + "\n  " +frase;
            tvFrase.setText(string);

        }

    }

}

File: AppSorteio/app/src/main/java/com/myo/appsorteio/MainActivity.kt
package com.myo.appsorteio

import android.annotation.SuppressLint
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.TextView

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }


    @SuppressLint("SetTextI18n")
    fun sortear(view: View){

        val numSorteado = findViewById<TextView>(R.id.tvNum)

        val num = java.util.Random().nextInt(11)

        numSorteado.setText("Número sorteado: "+ num)

    }

}